src/react/header/Header.tsx

```diff
@@ -0,0 +1,145 @@
+import clsx from 'clsx'
+import type { ReactNode } from 'react'
+import { useEffect, useRef } from 'react'
+import { MODlogo } from './MODlogo'
+
+interface NavigationOptions {
+  key?: string
+  content?: ReactNode | string
+  href?: string
+  active?: boolean
+  attributes?: Omit<JSX.IntrinsicElements['a'], 'href'>
```

Comment:
Just noticed that the global `JSX` is deprecated – it should be imported from `react` now.